recipes-tag/28-id-2-xpd-collection/meta.yaml

```diff
@@ -1,6 +1,6 @@
 {% set year = "2018" %}
 {% set cycle = "2" %}
-{% set version ="0" %}
+{% set version ="4" %}
```

Comment:
28-id-2 is not on cycle 3, same as 28-id-1?

Reply:
Hmm that must not have been updated?